samples/hello-oboe/src/main/cpp/jni_bridge.cpp

```diff
@@ -56,7 +56,7 @@ Java_com_google_sample_oboe_hellooboe_PlaybackEngine_native_1setToneOn(
         LOGE("Engine handle is invalid, call createHandle() to create a new one");
         return;
     }
-    engine->setToneOn(isToneOn);
+    engine->toggleTone();
```

Comment:
Again, not sure why this has changed to "toggling"

Reply:
See above.